make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -0,0 +1,40 @@
+/* TODO remove the table artifact_2 and use the artifact instead after finishing the upgrade work */
+CREATE TABLE artifact_2
+(
+  id            SERIAL PRIMARY KEY NOT NULL,
+  /* image, chart, etc */
+  type          varchar(255),
```

Comment:
I don't think we should store one semantic attribute in 2 columns

Reply:
It's easy for query. Thinking about listing only images, if we don't store the `type`, we need to get the corresponding media types from `Artifact Resolver` first(there are several media types for image), and use the sql something like `select * from artifact where media_type in ('media_type_1', 'media_type_2')` to query

And we need to define methods for `Artifact Resolver` to get type according to the media type and to get the media types according to type.

But if we store it, all these convert works can be avoid